packages/date-picker/test/basic.test.js

```diff
@@ -524,56 +416,35 @@ describe('ios', () => {
 
   it('should focus the input when closed', () => {
     datepicker.focus();
-    expect(isFocused(input)).to.be.true;
+    expect(document.activeElement).to.equal(input);
   });
 
   it('should blur the input when opened', async () => {
     datepicker.focus();
     await open(datepicker);
-    expect(isFocused(input)).to.be.false;
+    expect(document.activeElement).to.not.equal(input);
   });
 
   describe('auto open disabled', () => {
-    let toggleButton;
-
     beforeEach(() => {
       datepicker.autoOpenDisabled = true;
-      toggleButton = datepicker.shadowRoot.querySelector('[part="toggle-button"]');
     });
 
     it('should focus the input on touch tap', () => {
       touchTap(input);
-      expect(isFocused(input)).to.be.true;
+      expect(document.activeElement).to.equal(input);
     });
 
     it('should blur the input on open', async () => {
       touchTap(input);
       await open(datepicker);
-      expect(isFocused(input)).to.be.false;
-    });
-
-    it('should blur the input on fullscreen open', async () => {
-      datepicker._fullscreen = true;
-      touchTap(input);
-      await open(datepicker);
-      expect(isFocused(input)).to.be.false;
+      expect(document.activeElement).to.not.equal(input);
     });
 
     it('should not open on input tap', () => {
       tap(input);
       expect(datepicker.opened).not.to.be.true;
     });
-
-    it('should not open on input tap on fullscreen', () => {
-      datepicker._fullscreen = true;
-      tap(input);
-      expect(datepicker.opened).not.to.be.true;
-    });
-
-    it('should open by tapping the calendar icon even if autoOpenDisabled is true', async () => {
-      tap(toggleButton);
-      await oneEvent(datepicker.$.overlay, 'vaadin-overlay-open');
-    });
```

Comment:
Removed this duplicate test, because `autoOpenDisabled` has no effect on the toggle button.